src/components/cards/DomainCheckoutCard.tsx

```diff
@@ -0,0 +1,159 @@
+import { useCountdown } from '@src/hooks/useCountdown';
+import useDomainInfo from '@src/hooks/useDomainInfo';
+import { useWalletState } from '@src/state';
+import { isArweaveTransactionID } from '@src/utils';
+import { DEFAULT_ANT_LOGO, NETWORK_DEFAULTS } from '@src/utils/constants';
+import { RotateCw } from 'lucide-react';
+import { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+
+import ANTDetailsTip from '../Tooltips/ANTDetailsTip';
+import { Tooltip } from '../data-display';
+import { AntLogoIcon } from '../data-display/AntLogoIcon';
+import { ArNSLogo } from '../icons';
+
+export type DomainCheckoutCardProps = {
+  domain: string;
+  antId?: string;
+  targetId?: string;
+  orderSummary: Record<string, ReactNode>;
+  fees: Record<string, ReactNode>;
+  quoteEndTimestamp: number;
+  refresh: () => void;
+};
+
+function DomainCheckoutCard({
+  domain,
+  antId,
+  targetId,
+  orderSummary,
+  fees,
+  quoteEndTimestamp,
+  refresh,
+}: DomainCheckoutCardProps) {
+  const [{ walletAddress }] = useWalletState();
+  const { data: domainInfo } = useDomainInfo({
+    antId: isArweaveTransactionID(antId) ? antId : undefined,
+  });
+
+  const countdownString = useCountdown(quoteEndTimestamp);
+  return (
+    <div className="relative flex flex-col w-full h-full rounded border border-dark-grey overflow-hidden transition-all duration-300">
+      {/* domain detail card */}
+      <ArNSLogo className="absolute w-[320px] h-fit top-[0px] right-[0px]" />
+      <div className="flex flex-col size-full bg-gradient-to-b from-[#1C1C1F] to-[#0E0E0F] rounded-t text-light-grey p-6">
```

Comment:
Lot of RGB color values in this file, good to get keys to add to tailwind config

Reply:
[289529f](https://github.com/ar-io/arns-react/pull/731/commits/289529f47a0bba162e91327c58045781ea055538)

I stuck with the hex codes for the gradient because i couldnt figure out what to name the keys